src/agent/onefuzz-agent/src/local/tui.rs

```diff
@@ -229,6 +317,49 @@ impl TerminalUi {
                 .constraints([Constraint::Percentage(25), Constraint::Percentage(75)].as_ref())
                 .split(f.size());
 
+            let log_area = chunks[1];
+            let top_area = Layout::default()
+                .direction(Direction::Horizontal)
+                .constraints([Constraint::Percentage(50), Constraint::Percentage(50)].as_ref())
+                .split(chunks[0]);
+
+            let file_count_area = top_area[0];
+            let coverage_area = Layout::default()
```

Comment:
When I have a taller than average window (such as a 75 row terminal), the bar chart is 4 rows tall.  Is this intentional?